dev/breeze/src/airflow_breeze/utils/errors.py

```diff
@@ -0,0 +1,117 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
should be renamed to "docs_errors"  I think.

Reply:
Yeah makes sense. Renaming